src/utils/LogUtils.ts

```diff
@@ -1 +1,10 @@
+// Fix to console log in single line, and add error message to log.
```

Comment:
With this implementation, will error message also be added to the log?

Reply:
yes it will be something like 
```
{"args":{"_":[],"dataworker":false,"finalizer":false,"h":false,"help":false,"monitor":false,"relayer":true,"wallet":"secret"},"at":"relayer","bot-identifier":"NO_BOT_ID","e":{},"error":"Error: No RPC providers defined for chainId 1\n    at resolveUrls (/Users/lisk/Documents/10_lisk/across-relayer/dist/src/utils/ProviderUtils.js:570:19)\n    at getNodeUrlList (/Users/lisk/Documents/10_lisk/across-relayer/dist/src/utils/ProviderUtils.js:585:22)\n    at getProvider (/Users/lisk/Documents/10_lisk/across-relayer/dist/src/utils/ProviderUtils.js:542:38)\n    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at async constructClients (/Users/lisk/Documents/10_lisk/across-relayer/dist/src/common/ClientHelper.js:193:29)\n    at async constructRelayerClients (/Users/lisk/Documents/10_lisk/across-relayer/dist/src/relayer/RelayerClientHelper.js:31:27)\n    at async Object.runRelayer [as relayer] (/Users/lisk/Documents/10_lisk/across-relayer/dist/src/relayer/index.js:30:28)\n    at async run (/Users/lisk/Documents/10_lisk/across-relayer/dist/index.js:43:9)","level":"error","message":"There was an execution error!","notificationPath":"across-error","reason":{},"run-identifier":"655aa652-f7e6-426e-8d6b-436832a1314c"}
```